.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, shift-64622] # todo@1 remove shift-64622 before merge
```

Comment:
FYI per our TODO numbering system this would be `todo0` (since it should be resolved before merging) and we don't use `@` with numbers, only with groups